### PR TITLE
Add query for metrics on multi-allelic "multi-sample variants" data.

### PR DESCRIPTION
**Pull request by deflaux**

### CODE REVIEW

File: 1000genomes_phase3/README.Rmd [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+```{r init, echo=FALSE, message=FALSE, warning=FALSE, comment=NA}
+library(bigrquery)
+library(ggplot2)
+library(dplyr)
+library(xtable)
+```
+
+```{r echo=FALSE, message=FALSE, warning=FALSE}
+# TODO(deflaux) change to public tables before publishing.
```
Comment (mbookman) [inline 1000genomes_phase3/README.Rmd]: Do these need to be updated?

File: 1000genomes_phase3/README.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+### Additional Resources
+* [Schema](https://bigquery.cloud.google.com/table/genomics-public-data:1000_genomes_phase_3.variants_20150220_release?pli=1)
+* [Data Provenance](http://googlegenomics.readthedocs.io/en/latest/use_cases/discover_public_data/1000_genomes.html)
+* [Index of variant analyses](./sql)
+
+### Metrics
+
+In the following plots we display metrics computed on both 1,000 Genomes phase 1 and phase 3 variants.
+
```
Comment (mbookman) [inline 1000genomes_phase3/README.md]: Why so many blank lines?

File: 1000genomes_phase3/README.Rmd (hunk 1 above)
Comment (deflaux) [inline 1000genomes_phase3/README.Rmd]: removed, thanks!

File: 1000genomes_phase3/README.md (hunk 2 above)
Comment (deflaux) [inline 1000genomes_phase3/README.md]: knitr put those in.  This file is auto-generated.

In the markdown in this repo, I suppress most/all of the echoing of R code.

In the codelabs repo, I echo nearly all the R code.

**Comment by mbookman** [comment]
LGTM

File: 1000genomes_phase3/README.Rmd [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+  ylab("Ti/Tv Ratio") +
+  xlab("Dataset") +
+  ggtitle("Heterozygous/Homozygous Variant Ratio") 
+
+ggplot(results, aes(dataset, het_hom_ratio)) +
+  geom_boxplot(aes(colour = Gender)) +
+  ylab("Ti/Tv Ratio") +
+  xlab("Dataset") +
+  ggtitle("Heterozygous/Homozygous Variant Ratio") 
+```
```
Comment (pgrosu) [inline 1000genomes_phase3/README.Rmd]: Hi Nicole,

Something doesn't look quite right with some of the plots.  So if in your query you have the following definitions on [lines 11-12](https://github.com/deflaux/bigquery-examples/blob/b180ed78a5b7127d985961efc028e746fbfed7f5/1000genomes_phase3/sql/qc-metrics.sql#L11-12):

```
transitions_count / transversions_count AS ti_tv_ratio,
(het_RA_count + het_AA_count) / (hom_RR_count + hom_AA_count) AS het_hom_ratio,
```

Why are the y-axes labels then not matching in your ggplots's y-axis input variables for the following four plots:

```R
ggplot(results, aes(dataset, ti_tv_ratio)) +
  geom_boxplot(aes(colour = Super_Population)) +
  ylab("Het/Hom Ratio") +
  xlab("Dataset") +
  ggtitle("Transition/Transversion SNP Ratio") 

ggplot(results, aes(dataset, ti_tv_ratio)) +
  geom_boxplot(aes(colour = Gender)) +
  ylab("Het/Hom Ratio") +
  xlab("Dataset") +
  ggtitle("Transition/Transversion SNP Ratio") 

ggplot(results, aes(dataset, het_hom_ratio)) +
  geom_boxplot(aes(colour = Super_Population)) +
  ylab("Ti/Tv Ratio") +
  xlab("Dataset") +
  ggtitle("Heterozygous/Homozygous Variant Ratio") 

ggplot(results, aes(dataset, het_hom_ratio)) +
  geom_boxplot(aes(colour = Gender)) +
  ylab("Ti/Tv Ratio") +
  xlab("Dataset") +
  ggtitle("Heterozygous/Homozygous Variant Ratio") 

```

  ~p

Comment (deflaux) [inline 1000genomes_phase3/README.Rmd]: good catch!  https://github.com/googlegenomics/bigquery-examples/pull/35

Comment (pgrosu) [inline 1000genomes_phase3/README.Rmd]: Cool, thanks for the fix :)

~p

**Pull request merged**